trunk/conf/full.conf

```diff
@@ -1460,6 +1460,14 @@ vhost hls.srs.com {
         # the key root URL, use this can support https.
         # @remark It's optional.
         hls_key_url       https://localhost:8080;
+	# Whether enable hls_ctx.
+	# hls_ctx used to keep track of subsequent m3u8 requests.
+	# if on, hls client info can be collected in stat module,
+	# if on, on_play/on_stop can be called in http hook module.
+	# otherwise not;
+	# When request stopped, http hook or stat with special time effected, that is 2*hls_window, 
+	# Default: off
+	hls_ctx		on;
```

Comment:
It says default off in above line, but value is `on` in this line. Suggest change the value as `off`.

Reply:
yeah, I will modifie it.